QEfficient/cloud/infer.py

```diff
@@ -106,10 +120,13 @@ def main(
             full_batch_size=full_batch_size,
         )
 
+        compile_time = (time.perf_counter() - compile_start_time) // 1
```

Comment:
This can also be moved under the if condition of benchmark flag, also why keeping it integer?